ethereum/core/src/test/java/org/hyperledger/besu/ethereum/vm/OperandStackTest.java

```diff
@@ -83,17 +85,17 @@ public void get() {
     assertThat(stack.get(2)).isEqualTo(Bytes32.fromHexString("0x01"));
   }
 
-  @Test(expected = UnderflowException.class)
+  @Test
   public void set_NegativeOffset() {
     final OperandStack stack = new OperandStack(1);
-    stack.get(-1);
+    assertThatThrownBy(() -> stack.get(-1)).isInstanceOf(UnderflowException.class);
```

Comment:
we have two tests doing the same thing - should this one be stack.set()
see get_NegativeOffset

Reply:
super catch! will fix them

Reply:
does it make sense to fix those 2 tests in a separate PR?

Reply:
I was thinking on adding a first commit with just those changes, but if you think it's better to put it separately to gain more attention I can do it

Reply:
A separate commit is probably fine, it's updating tests only. If the new tests fail then that's a different story 😱

Reply:
I had to change the exception type for the test b/c _get_ was throwing `IndexOutOfBoundsException` while _set_ throws `IndexOutOfBoundsException`

Reply:
👀